### PR TITLE
Removing unused libraries from AwsCrtClientCallingPatternIntegrationTest and update ThreadCount setting for test cases in aws-crt-client

### PR DESCRIPTION
**Pull request by joviegas**
## Motivation and Context
- Presence of  unused package imports in AwsCrtClientCallingPatternIntegrationTest was affecting integ test.

## Modifications

- - Removed unused package imports cased from AwsCrtClientCallingPatternIntegrationTest
- 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/aws-sdk-java-v2/blob/master/CONTRIBUTING.md) document
- [ ] Local run of `mvn install` succeeds
- [ ] My code follows the code style of this project
- [ ] My change requires a change to the Javadoc documentation
- [ ] I have updated the Javadoc documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] I have added a changelog entry. Adding a new entry must be accomplished by running the `scripts/new-change` script and following the instructions. Commit the new file created by the script in `.changes/next-release` with your changes.
- [ ] My change is to implement 1.11 parity feature and I have updated [LaunchChangelog](https://github.com/aws/aws-sdk-java-v2/blob/master/docs/LaunchChangelog.md)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

### CODE REVIEW

File: http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java [hunk 1]
```diff
@@ -113,17 +121,20 @@ private boolean testWithNewClient(int eventLoopSize, int numberOfRequests) {
...
     @DataPoints("EventLoop")
     public static int[] eventLoopValues(){
+        log.warn(() -> "eventLoopValues");
         return new int[]{1, 4};
     }
 
     @DataPoints("ConnectionPool")
     public static int[] connectionsValues(){
         /* Don't use 1 connection Pool of size 1, otherwise test takes too long */
+        log.warn(() -> "connectionsValues");
```
Comment (cenedhryn) [inline http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java:131]: Is this log line necessary to keep after the bug is fixed?

File: http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java [hunk 2]
```diff
@@ -197,8 +213,14 @@ public void checkAllCombinations(@FromDataPoints("EventLoop") int eventLoopSize,
...
             float numSeconds = (float) ((System.currentTimeMillis() - start) / 1000.0);
+            float numSeconds_test = (float) ((System.currentTimeMillis() - start_test) / 1000.0);
+
             String timeElapsed = String.format("%.2f sec", numSeconds);
 
+            String timeElapsed_test = String.format("%.2f sec", numSeconds_test);
+
+            log.warn(() -> "Passed: " + testName + ", Time " + timeElapsed_test);
+
             System.out.println("Passed: " + testName + ", Time " + timeElapsed);
```
Comment (cenedhryn) [inline http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java:224]: We can remove the println statement or move it to log

File: http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java [hunk 3]
```diff
@@ -197,8 +213,14 @@ public void checkAllCombinations(@FromDataPoints("EventLoop") int eventLoopSize,
             CrtResource.waitForNoResources();
 
             float numSeconds = (float) ((System.currentTimeMillis() - start) / 1000.0);
+            float numSeconds_test = (float) ((System.currentTimeMillis() - start_test) / 1000.0);
```
Comment (cenedhryn) [inline http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java:216]: It's confusing with two different timing statements, the numSeconds and numSeconds_test. Do we need both? If not, which makes most sense? If we do can we refactor to make it clearer?

**Review by cenedhryn — COMMENTED** [review]
I'm (very) curious, how did the troubleshooting show that the import statements were affecting the results?

File: http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java (hunk 1 above)
Comment (joviegas) [reply http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java]: Had added for debugging , removed them as they are no longer required

File: http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java (hunk 3 above)
Comment (joviegas) [reply http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java]: this was added for testing integ test trigerred by this PR , now its no longer required

**Comment by joviegas** [comment]
> I'm (very) curious, how did the troubleshooting show that the import statements were affecting the results?

The test cases are getting stuck even before the Test classes are loaded. Confirmed this by
1. Adding logs in static blocks and static methods ,, there logs are not printed.
2. Removed @Theory and executed the test and it passed.

Thus it looks like there is some issue in maven-surefire-plugin/maven while it prepares the test cases.


I also tried some setting and changing threadCountClasses to caused the test case to pass, below is documentation for threadCountClasses
https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html

**Review by cenedhryn — APPROVED** [review]
Let's try it!

**Pull request merged**